# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/trusted-types/block-eval-function-constructor.html.ini
# Commit: df960f6a3f76
# Full Hash: df960f6a3f76a3fc92fe57054830649c15f4ff3a
# Author: Frédéric Wang <fwang@igalia.com>
# Date: 2025-01-17 21:57:54
# Regressor Bug: 1901492
# File Overlap Count: 1
# Description:
#   Bug 1901492 - Make GetTrustedTypesCompliantString work with workers. r=smaug,tschuster
#   
#   Make sure GetTrustedTypesCompliantString() work when the specified
#   global object is a WorkerGlobalScope. If that happens the function
#   is executed from the worker thread. The following changes are made:
# ==============================================================================

diff -r c6296179fcb4 -r df960f6a3f76 testing/web-platform/meta/trusted-types/block-eval-function-constructor.html.ini
--- a/testing/web-platform/meta/trusted-types/block-eval-function-constructor.html.ini	Fri Jan 17 13:35:40 2025 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,36 +0,0 @@
-[block-eval-function-constructor.html]
-  [Blocked eval in DedicatedWorkerGlobalScope.]
-    expected: FAIL
-
-  [Blocked indirect eval in DedicatedWorkerGlobalScope.]
-    expected: FAIL
-
-  [Blocked Function constructor in DedicatedWorkerGlobalScope.]
-    expected: FAIL
-
-  [Blocked AsyncFunction constructor in DedicatedWorkerGlobalScope.]
-    expected: FAIL
-
-  [Blocked GeneratorFunction constructor in DedicatedWorkerGlobalScope.]
-    expected: FAIL
-
-  [Blocked AsyncGeneratorFunction constructor in DedicatedWorkerGlobalScope.]
-    expected: FAIL
-
-  [Blocked eval in SharedWorkerGlobalScope.]
-    expected: FAIL
-
-  [Blocked indirect eval in SharedWorkerGlobalScope.]
-    expected: FAIL
-
-  [Blocked Function constructor in SharedWorkerGlobalScope.]
-    expected: FAIL
-
-  [Blocked AsyncFunction constructor in SharedWorkerGlobalScope.]
-    expected: FAIL
-
-  [Blocked GeneratorFunction constructor in SharedWorkerGlobalScope.]
-    expected: FAIL
-
-  [Blocked AsyncGeneratorFunction constructor in SharedWorkerGlobalScope.]
-    expected: FAIL